packages/statemanager/src/stateManager.ts

```diff
@@ -948,27 +948,36 @@ export class DefaultStateManager implements EVMStateManagerInterface {
    * at the last fully committed point, i.e. as if all current
    * checkpoints were reverted.
    *
-   * Note on caches:
+   * Caches are downleveled (so: adopted for short-term usage)
+   * by default.
+   *
+   * This means in particular:
    * 1. For caches instantiated as an LRU cache type
    * the copy() method will instantiate with an ORDERED_MAP cache
    * instead, since copied instantances are mostly used in
    * short-term usage contexts and LRU cache instantation would create
    * a large overhead here.
-   * 2. Cache values are generally not copied along
+   * 2. The underlying trie object is initialized with 0 cache size
+   *
+   * Both adoptions can be deactivated by setting `downlevelCaches` to
+   * `false`.
+   *
+   * Cache values are generally not copied along.
```

Comment:
What does this comment mean in context now?  It feels imprecise and doesn't make it clear in which circumstances (if any) the cache values actually would be copied.

Would it be better to say: `The underlying trie objet is initialized with 0 cache size and cache values are not copied over?`

Reply:
Hmm, this does not only count for point 2.

I have adopted locally to: "Cache values are generally not copied along regardless of the `downlevelCaches` setting."

Will push soon.